### PR TITLE
fix: Grab all latest transactions

### PR DESCRIPTION
**Pull request by johnletey**
As we discovered, the way we were monitoring the wallet, meant that the trading post would drop transactions.

I have created a fix for this 🥳

**Note:** We'll want to set the constants back to the production ones before merging this 😄

### CODE REVIEW

**Review by divy-work — APPROVED** [review]
Great job! :rainbow:

File: src/utils/constants.yml [hunk 1]
```diff
@@ -1,2 +1,5 @@
-exchangeContractSrc: fE2OcfjlS-sHqG5K8QvxE8wHtcqKxS-YV0bDEgxo-eI
-exchangeWallet: aLemOhg9OGovn-0o4cOCbueiHT9VgdYnpJpq7NgMA1A
+exchangeContractSrc: c25-RdheC6khcACLv23-XXg1W7YuA-VSZ_1_qnNFbhw
```
Comment (t8) [inline src/utils/constants.yml:1]: We're going to need to change this back to the production contract.

File: src/utils/constants.yml [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+exchangeWallet: pvPWBZ8A5HLpGSEfhEmK1A3PfMgB_an8vVS6L14Hsls
```
Comment (t8) [inline src/utils/constants.yml:2]: Going to need to change this back to production exchange address.

**Review by t8 — CHANGES_REQUESTED** [review]
This is great! Just need to revert to prod contracts and this is ready to merge!

**Pull request merged**